tests/features/common/kie-controller-common.feature

```diff
@@ -30,9 +39,19 @@ Feature: KIE Controller configuration common tests
      And container log should contain External authentication/authorization enabled, skipping the embedded users creation.
      And container log should contain KIE_ADMIN_USER is set to customAdm, make sure to configure this user with the provided password on the external auth provider with the roles kie-server,rest-all,admin,kiemgmt,Administrators
 
-  Scenario: Check if eap users are not being created if LDAP is configured
+  Scenario: Check if eap users are not being created if LDAP is configured and EXTERNAL_AUTH_ONLY is not set
+    When container is started with env
+      | variable                   | value         |
+      | AUTH_LDAP_URL              | ldap://url:389|
+      | KIE_ADMIN_USER             | customAdm     |
+      | KIE_ADMIN_PWD              | custom" Adm!0 |
+    Then file /opt/eap/standalone/configuration/application-users.properties should contain customAdm=a4d41e50a4ae17a50c1ceabe21e41a80
+     And file /opt/eap/standalone/configuration/application-roles.properties should contain adminUser=kie-server,rest-all,admin,kiemgmt,Administrators,user
+
+  Scenario: Check if eap users are not being created if LDAP is configured and EXTERNAL_AUTH_ONLY is set to true
```

Comment:
For this scenario it should be also good to check also log that contains info from print_external_user_information()

Reply:
See lines 60-61